README.md

```diff
@@ -1,23 +1,46 @@
+## Write once, run anywhere, connect everyone 
+
 
 ### Description
 
-Socket is a client-side runtime for creating native cross-platform software on
-mobile and desktop using HTML, CSS, and JavaScript. It also exposes primitives
-needed for building peer-to-peer and local-first applications, such as Bluetooth,
-UDP and File I/O.
+`Socket Runtime`. provides a range of tools and software utilities to help developers build powerful application experiences using standard web technologies `(HTML, CSS, and JS)`. These applications are compiled by Socket's tools into hybrid-native applications -- meaning a combination of web code running in a platform's "WebView" (embedded browser) along with platform-native code: Java on Android, Swift/Objective-C on iOS, C++ on Windows or Linux, etc.
+
+These compiled applications can be distributed through major application stores, such as Google Play (Android, ChromeOS), App Store (iOS, MacOS), and Microsoft Store (Windows). These apps can thus be distributed and run on all major consumer computing platforms: Windows, MacOS, Linux, iOS, and Android.
+
+### Features
+
+* Local First, a full featured File system API & Bluetooth.
+* P2P & Cloud, built to support a new generation of apps that can connect directly to each other by providing a high performance UDP API.
+* Use any backend.
+* Use any frontend.
+* Maintainable, zero dependencies and a smaller code base than any other competing project.
+* Lean & Fast, uses a smaller memory footprint and creates smaller binaries than any other competing project.
+
+### FAQ
+
+Check the [Website]((https://sockets.sh/) to learn more.
+ <!--we can Kyle's FAQ here, I read it yesterday and it's perfect  -->
+
+ 
+### Building your first Socket app !
+
+If you are interested in making an `Socket` app, please check [Create Socket App Repo](https://github.com/socketsupply/create-socket-app).  
+`Examples` can be found in the [Examples Repo](https://github.com/socketsupply/socket-examples).  
+
 
 ### Documentation
 
-<!-- [![Socket SDK CI](https://github.com/socketsupply/socket/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/socketsupply/socket/actions/workflows/ci.yml) -->
+The full documentation for `Socket Runtime` can be found on our [Website](https://sockets.sh/)  
+The `Socket Runtime` documentation discusses API, examples , multiple guides (Apple,Desktop and Windows), P2P section and more.
 
-Please read (and contribute to) the documentation [here](https://sockets.sh).
 
 ### Testing
 
-See the [`test/`](test/) for the unit and integration test suite.
+Test all the things! See the [`test/`](test/) for the unit and integration test suite.
 
-### Development
+### Contributing
 
-If you want to contribute to the Socket Runtime project itself, please connect
-with any current project contributor.
+Please stop by [Discord](https://discord.com/invite/YPV32gKCsH) and ask the team about your issue and if someone is already working on it.
```

Comment:
We can think about publishing install.sh and install.ps1 instructions now. These are used for building the CLI and app libs.

Reply:
>If you want to contribute to the Socket Runtime project itself, please connect
with any current project contributor.

I think we can have a detailed `contribution Guide` in the near future, I liked how many other websites have that. This guide can be a link in `readme` or a `detailed section` in the docs.